Sources/dep/Git.swift

```diff
@@ -43,9 +43,8 @@ class Git {
         }(self)
 
         var versions: [Version] {
-            //TODO separator is probably \r\n on Windows
             let out = (try? popen([Git.tool, "-C", root, "tag", "-l"])) ?? ""
-            let tags = out.characters.split("\n")
+            let tags = out.characters.split { $0 == "\n" || $0 == "\r\n" }.map(String.init)
```

Comment:
Apple has had a blog post about how strings work in Swift 2.0. Worth a read :) https://developer.apple.com/swift/blog/?id=30